core/src/codecs/snapshot/SSStructuredIndexFormat.cpp

```diff
@@ -127,7 +127,7 @@ SSStructuredIndexFormat::read_internal(const milvus::storage::FSHandlerPtr& fs_p
 
     double span = recorder.RecordSection("End");
     double rate = length * 1000000.0 / span / 1024 / 1024;
-    LOG_ENGINE_DEBUG_ << "read_index(" << path << ") rate " << rate << "MB/s";
+    LOG_ENGINE_DEBUG_ << "SSStructuredIndexFormat::read(" << location << ") rate " << rate << "MB/s";
```

Comment:
Macro "LOG_ENGINE_DEBUG_" will output this function name to log, so why we need this change